src/components/DiscordPage/index.tsx

```diff
@@ -4,10 +4,11 @@ import Iframe from 'react-iframe';
 import './style.less';
 
 const DiscordPage = () => {
+  const ReactIframe = Iframe as React.ComponentClass<any>;
   return (
     <div className="discord-page">
       <h1 className="title">Discord</h1>
-      <Iframe
+      <ReactIframe
```

Comment:
If I recall, the `react-iframe` package has an interface for its iframe, `IIframe`. Writing out locally, I find some fields from the properties of this component are not permitted in `react-iframe`. (`display`, `allowtransparency`, `frameborder`)